ci/concourse/oss/rpmbuild.py

```diff
@@ -99,6 +97,10 @@ def rpm_package_name(self):
     def _prepare_rpm_build_dir(self):
         for sub_dir in ["SOURCES", "SPECS"]:
             os.makedirs(os.path.join(self.rpm_build_dir, sub_dir), mode=0o755)
+        
+        if self._platform == 'sles11':
+            for sub_dir in ["BUILD", "RPMS"]:
+                os.makedirs(os.path.join(self.rpm_build_dir, sub_dir), mode=0o755)
```

Comment:
Do we need to specify sles11 here or can the function attempt to make all the needed subdirs without checking platform.

```suggestion
    def _prepare_rpm_build_dir(self):
        for sub_dir in ["SOURCES", "SPECS","BUILD","RPMS"]:
            os.makedirs(os.path.join(self.rpm_build_dir, sub_dir), mode=0o755)        
```

Reply:
Done